src/components/UserInfo.js

```diff
@@ -1,16 +1,27 @@
+import { api } from "../pages/index.js";
+
 class UserInfo {
-  constructor(nameSelector, jobSelector) {
-    this._name = document.querySelector(nameSelector);
-    this._job = document.querySelector(jobSelector);
+  constructor(userData) {
+    this._name = document.querySelector(".profile__name");
+    this._name.textContent = userData.name;
+    this._about = document.querySelector(".profile__about");
+    this._about.textContent = userData.about;
+    this._avatar = document.querySelector(".profile__image");
+    this._avatar.src = userData.avatar;
```

Comment:
`nameSelector, jobSelector` should be used here and `avatarSelector` as well not to hardcode them inside the class according to the task

Reply:
Sorry I can't find this requirement anywhere in the project... I understand that it would make it a bit more robust when they are declaring the different class instances and declaring the different Selectors. So I guess this Class would end up with 4 elements to utilized i.e. (nameSelector, jobSelector, avatarSelector, data)?

Reply:
I updated it to reflect the changes wanted.